addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -366,18 +395,8 @@ export class WebsocketWorker {
     }
 
     _logDebug(title, ...args) {
-        const clientsInDebug = [...this.debugModeByClient.keys()].filter((client) =>
-            this.debugModeByClient.get(client)
-        );
-        for (const client of clientsInDebug) {
-            client.postMessage({
-                type: "log_debug",
-                data: [
-                    `%c${new Date().toLocaleString()} - [${title}]`,
-                    "color: #c6e; font-weight: bold;",
-                    ...args,
-                ],
-            });
+        if (this.loggingEnabled) {
+            logger.log({ dt: new Date().toISOString(), event: title, args, worker: UUID });
```

Comment:
Could serialize `dt` to have same datetime format as in payload that uses server format.